lightly/loss/tico_loss.py

```diff
@@ -0,0 +1,80 @@
+import torch
+import torch.distributed as dist
+import torch.nn.functional as F
+from typing import Tuple
+
+from lightly.utils.dist import gather
+
+class TiCoLoss(torch.nn.Module):
+    """Implementation of the Tico Loss from Tico[0] paper.
+    This implementation takes inspiration from the code published 
+    by sayannag using Lightly. [1]
+
+    [0] Jiachen Zhu et. al, 2022, Tico... https://arxiv.org/abs/2206.10698
+    [1] https://github.com/sayannag/TiCo-pytorch
+        
+    Examples:
+    
+        >>> # initialize loss function
+        >>> loss_fn = TiCoLoss()
+        >>>
+        >>> # generate two random transforms of images
+        >>> t0 = transforms(images)
+        >>> t1 = transforms(images)
+        >>>
+        >>> # feed through model
+        >>> out0, out1 = model(t0, t1)
+        >>>
+        >>> # calculate loss
+        >>> loss = loss_fn(out0, out1)
+    """
+
+    def __init__(
+        self,
+        beta_param: float = 0.9,
+        ro_param: float = 20.0,
+        gather_distributed : bool = False,
+    ):
+        """Lambda, mu and nu params configuration with default value like in [0]
+        Args:
```

Comment:
Where do lambda, mu, and nu come from? I cannot find them in the paper.

Could you also add a newline before `Args:`?